src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+@import "../utils/variables";
```

Comment:
when you @import variables you should not do it to each file, it is enough to @import everything to main.scss file
but when you @use, then use it in every file you need

Reply:
When I use @import in another file, I have an error, so I need to include _variables.scss in each file where needed